tools/delete_candidate.php

```diff
@@ -26,264 +41,195 @@
-foreach ($instruments as $instrument) {
-
-    //delete the entry from the instrument table
-    $DB->delete(
-        $instrument['Test_name'], array("CommentID" => $instrument['CommentID'])
-    );
-
-    //delete from flag
-    $DB->delete("flag", array("ID" => $instrument['ID']));
-    
-     //delete from conflicts_resolved
-    $DB->delete(
-        "conflicts_resolved", array("CommentId1" => $instrument['CommentID'])
-    );
-    $DB->delete(
-        "conflicts_resolved", array("CommentId2" => $instrument['CommentID'])
-    );
-    //delete from conflicts_unresolved
-    $DB->delete(
-        "conflicts_unresolved", array("CommentId1" => $instrument['CommentID'])
-    );
-    $DB->delete(
-        "conflicts_unresolved", array("CommentId2" => $instrument['CommentID'])
-    );
-    
-    //delete from final_radiological_review
-    $DB->delete(
-        "final_radiological_review", 
-               array("CommentID" => $instrument['CommentID'])
-    );
-    
+
+function deleteCandidate($CandID, $PSCID, $confirm, $printToSQL, $DB, $output)
+{
+
+    //Find candidate...
+    $candidate = new Candidate();
+    $candidate->select($CandID); //find the candidate with the given CandID
+
+    // Passing argument to delete session script
+    $outputType ="";
+    if ($printToSQL) {
+        $outputType ="tosql";
+    } else if ($confirm) {
+        $outputType ="confirm";
+    }
+
+    //find sessions
+    $sessions = $candidate->getListOfTimePoints();
+    if (is_null($sessions) || empty($sessions)) {
+        echo "There are no corresponding sessions for CandID: $CandID \n";
+    } else {
+        foreach ($sessions as $sid) {
+            $out = shell_exec(
+                "php ".__DIR__."/delete_timepoint.php delete_timepoint".
+                " $CandID $PSCID $sid $outputType"
+            );
+            echo $out;
+        }
+    }
+
+    // Print participant_status
+    echo "\nParticipant Status\n";
+    echo "--------------------\n";
+    $result = $DB->pselect('SELECT * FROM participant_status WHERE CandID=:cid', array('cid' => $CandID));
+    print_r($result);
+
+    // Print participant_status_history
+    echo "\nParticipant Status History\n";
+    echo "----------------------------\n";
+    $result = $DB->pselect('SELECT * FROM participant_status_history WHERE CandID=:cid', array('cid' => $CandID));
+    print_r($result);
+
+    // Print parameter_candidate
+    echo "\nParameter Candidate\n";
+    echo "---------------------\n";
+    $result = $DB->pselect('SELECT * FROM parameter_candidate WHERE CandID=:cid', array('cid' => $CandID));
+    print_r($result);
+
+    // Print candidate
+    echo "\nCandidate\n";
+    echo "-----------\n";
+    $result = $DB->pselect('SELECT * FROM candidate WHERE CandID=:cid', array('cid' => $CandID));
+    print_r($result);
+
+    // IF CONFIRMED, DELETE CANDIDATE
+    if ($confirm) {
+        echo "\nDropping all DB entries for candidate CandID: " . $CandID . "And PSCID: " .
```

Comment:
Super duper incredibly minor detail, but a space could be added before the "And PSCID: " string, so: " And PSCID: " 
Same for line 195.